web/components/Canvas/Sidebar/Chat/Chat.jsx

```diff
@@ -30,13 +44,16 @@ function Chat() {
       });
     }
   }, [websocket]);
-
+  // TODO: action history isn't right..updates every time somebody chats

```

Comment:
Tbh I don't think we really need to preserve the action history. That was more of a feature for the demo imo